site/content/pages/contact.md

```diff
@@ -5,19 +5,19 @@ menu:
   main:
     parent: aboutmenu
 ---
-Shift attempts to be a transparent organization.  Most of the organizational work for the organization is done at the monthly board meetings which you are welcome to attend.  We post our [meeting minutes](/pages/meeting-minutes) on this website if you don't want to sit with us for a couple hours.
+## Want to connect with Shift?
 
-If you want to get in touch with the board as a whole, [you can email the board of directors here](mailto:shift-bod@googlegroups.com).  If you want to get in touch with the Shift community, most general discussion about the community and events takes place on the [mailing list](/pages/email-list/) which you're encouraged but not required to join.
+- [Email our Board of Directors](mailto:shift-bod@googlegroups.com)
+- [Email our Website and Calendar Crew](mailto:bikecal@shift2bikes.org)
+- [Join our Community Mailing list](https://www.shift2bikes.org/pages/email-list/)
```

Comment:
Could relative URL. Still works, but will also work in local dev if relative URL. 
```suggestion
- [Join our Community Mailing list](/pages/email-list/)
```